worker/pb/api.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.0
-// 	protoc        v3.20.1
+// 	protoc        v3.17.3
```

Comment:
Not sure about the difference, probably nothing changed, but you might want to upgrade to 3.20.x

Reply:
Updated. Unfortunately, on MacOS the highest available version is v3.19.4.